### ISSUE TITLE
All the commands on the Extra > Mixer menu change the volume incorrectly

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#3608]
### Bug description

All the commands on the Extra > Mixer menu assume a linear volume scale which is then incorrectly converted to db.

### Steps to reproduce

For example:
1. Extra > Mixer > Adjust Playback Volume.
2. Enter 0.5 in the text box, and press OK.

### Expected behavior

The slider in the playback toolbar has the value -6 db.

### Actual behavior

The slider in the playback toolbar has the value -30 db

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

### Additional context

This is a regression wrt Audacity 3.1.3.
The sliders in the mixer/meter toolbars are not accessible for keyboard users:
https://github.com/audacity/audacity/issues/3230

**Comment by DavidBailes** [comment]
When I logged this bug I hadn't realized a number of things: that these commands do work on a log scale, just like they did in 3.1.3; the scales in the combined meter/mixer toolbars are wrong for the sliders.
So I'm going to close this bug. When the problem with the scales in these toolbars is resolved, I'll have another look at these commands.
